languages/en.php

```diff
@@ -154,6 +154,7 @@
 	'access:write' => "Write access",
 	'access:admin_only' => "Administrators only",
 	'access:missing_name' => "Missing access level name",
+	'access:comments:change' => "There are comments on this content, by changing the access level you will also be changing the access of those comments.  Do you want to continue?",
```

Comment:
Plus' language for this is: "This post was originally shared privately – be thoughtful about who you reshare it with. You can't reshare it publicly."

We could do: "This discussion is visible to a limited audience. Be thoughtful about who you share it with."